sky/cli.py

```diff
@@ -3077,6 +3077,9 @@ def _output():
                 raise click.UsageError(
                     f'Invalid accelerator quantity {accelerator_split[1]}. '
                     'Expected an integer.') from invalid_quantity
+            if quantity == 0:
+                raise click.UsageError('Invalid accelerator quantity 0. '
+                                       'Expected a non-zero integer.')
```

Comment:
nit: Instead of doing this check here, we can also implement this at L3075 inside the `try` block. That way we can reduce code duplication:
```
try:
    quantity = int(accelerator_split[1])
    if (quantity <= 0):
       raise ValueError('Quantity cannot be non-positive integer.')
except ValueError as invalid_quantity:
                raise click.UsageError(
                    f'Invalid accelerator quantity {accelerator_split[1]}. '
                    'Expected a non-zero integer.') from invalid_quantity
```

Note that the str in the ValueError does not matter much since we raise a click.UsageError later.

Reply:
This is neater! Thanks